0032-peering-clearing-settlement/0032-peering-clearing-settlement.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Peering, Clearing and Settling on the Interledger network
+draft: 1
+---
+
+The Interledger network is a graph of nodes (connectors) that have peered with one another by establishing a means of exchanging ILP packets and a means of paying one another for the successful forwarding and delivery of the packets.
+
+Further details on the ILP Protocol can be found in the [specification](../0027-interledger-protocol-4/0027-interledger-protocol-4.md).
+
+# Accounts and Balances
+
+The edge between any two nodes (peers) is a communication link (for exchanging ILP packets and other data), and an account held between the peers (the Interledger account). The Interledger account has a balance denominated in a mutually agreed upon asset (e.g. USD) at an agreed upon scale (e.g. 2). The balance on the account is **the net total of the amounts on any packets successfully routed between the peers**.
+
+The balance on the Interledger account can only change as a result of two events:
+ 1. The successful routing of an ILP packet between the two peers
```

Comment:
Might be worth mentioning here what successful means, or mentioning that it'll be explained below